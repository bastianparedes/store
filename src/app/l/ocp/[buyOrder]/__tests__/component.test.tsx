import React from 'react';

import { render } from '@testing-library/react';

import Component from '../component';

const buy = {
  buyerEmail: 'buyer@test.com',
  date: new Date(1702249134816),
  id: 'id-buy-1',
  productSku: 1
};

const product = {
  description: 'description',
  name: 'name',
  ownerEmail: 'seller@test.com',
  price: 10000,
  quantity: 5,
  sku: 1
};

it('should render', () => {
  const { container } = render(<Component buy={buy} product={product} />);
  expect(container).toMatchSnapshot();
});
