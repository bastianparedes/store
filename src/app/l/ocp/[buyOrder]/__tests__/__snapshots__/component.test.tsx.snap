// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Confirmación de pago
    </h1>
    <h2>
      Detalles de la orden
    </h2>
    <p>
      <strong>
        Orden de compra:
      </strong>
       
      id-buy-1
    </p>
    <p>
      <strong>
        Cantidad pagada:
      </strong>
       
      10000
    </p>
    <p>
      <strong>
        Fecha de compra:
      </strong>
       
      10/12/2023
    </p>
    <p>
      <strong>
        Email del comprador:
      </strong>
       
      buyer@test.com
    </p>
    <p>
      <strong>
        Email del vendedor:
      </strong>
       
      seller@test.com
    </p>
    <p
      style="align-items: center; display: flex;"
    >
      <strong>
        Evalúa al vendedor:
      </strong>
      <span
        style="color: gray; cursor: pointer; font-size: 2rem;"
      >
        ★
      </span>
      <span
        style="color: gray; cursor: pointer; font-size: 2rem;"
      >
        ★
      </span>
      <span
        style="color: gray; cursor: pointer; font-size: 2rem;"
      >
        ★
      </span>
      <span
        style="color: gray; cursor: pointer; font-size: 2rem;"
      >
        ★
      </span>
      <span
        style="color: gray; cursor: pointer; font-size: 2rem;"
      >
        ★
      </span>
    </p>
    <p>
      <strong>
        SKU del producto:
      </strong>
       
      1
    </p>
    <p>
      <strong>
        Nombre del producto:
      </strong>
       
      name
    </p>
    <p>
      <strong>
        Descripción del producto:
      </strong>
       
      description
    </p>
    <p>
      <strong>
        Imagen del producto:
      </strong>
    </p>
    <img
      alt="name"
      src="/api/image/1"
      style="max-width: 300px;"
    />
  </div>
</div>
`;
